Cargo.toml

```diff
@@ -58,7 +58,7 @@ serde_yaml = "0.9.19"
 
 [lib]
 path="src/lib.rs"
-name = "zenohc"
+name = "zenohcd"
```

Comment:
Is this on purpose?

Reply:
This was upstream of the PR, but I do wonder...

Reply:
Actually, probably just a typo that slipped in, removed it

Reply:
It seems that this change was re-pulled in...